.github/workflows/gitpod-web-docker.yml

```diff
@@ -1,12 +1,13 @@
 name: Gitpod-Web Docker
 on:
   workflow_dispatch:
+  push:
```

Comment:
Not sure about this, I'd say let's keep it manual, I can just do some change for gitpod-remote and this will rebuild a new image unnecessarily, also now that we decouple it, we want to bump the version manually each time  we want a new image like we do with gitpod-desktop extension so we can easily know which version is running in a workspace

Reply:
I see where you stand, @jeanp413. I still think this could really be helpful when working on features inside of Pull Requests, because you don't have to go trigger an action run (even harder when triggering from pull requests) - it's just one less step when developing.

It is true this will produce many unhelpful images, but I don't think that's too bad of a sacrifice when taking development velocity into account. Not _that many_ commits happen / will happen in this repo (just my guess though 😆).

Your call though, I'm reverting the change and we can introduce it in the future if we agree on it 🧡!

Reply:
I'd expect most of development can be tested/debugged just using the [launch configuration](https://github.com/gitpod-io/gitpod-code/blob/e041480c289c46422d407ae191f7943604b6506c/.vscode/launch.json#L7-L34) but that's a guess too :slightly_smiling_face: 

We can reconsider in the future for sure if that's not the case